uplink/src/base/mod.rs

```diff
@@ -64,6 +65,7 @@ pub struct Config {
     pub actions: Vec<String>,
     pub persistence: Option<Persistence>,
     pub streams: HashMap<String, StreamConfig>,
+    pub flush_period: u64,
```

Comment:
With this change, there won't be a way to use global timeout in case of it not being set in the config file, it would instead get defaulted to the 60s value.

Reply:
Also if we use `Duration`, given how it is deserialized, we will have to set the values in the config as follows: 
```
[streams.can]
topic = "/tenants/matter/devices/1106/events/can/jsonarray"
buf_size = 2
[streams.can.flush_period]
secs = 60
nanos = 0
```

Reply:
Getting defaulted to 60 is the behaviour we want. There is no use case for allowing users to change global default timeout